demo/read-input.js

```diff
@@ -1,16 +1,19 @@
-window.__i10c = {};
+window.__i10c = {
+    elements: document.getElementsByTagName('input'),
 
-window.__i10c.getInputValue = (inputEle) => {
-    return inputEle.value;
-}
+    getInputValue: function(inputEle) {
+        return inputEle.value;
+    },
 
-window.__i10c.setInputValue = (inputEle, val) => {
-    inputEle.value = val;
-}
+    setInputValue: function(inputEle, val) {
+        inputEle.value = val;
+    },
 
-window.__i10c.getAllInputValues = () => {
-    var elements = document.getElementsByTagName('input');
-    for (var i = 0; i < elements.length; i++) {
-        console.log("Found " + elements[i].value);
+    getAllInputValues: function() {
+        for (var i = 0; i < this.elements.length; i++) {
+            console.log("Found " + this.elements[i].value);
+        }
     }
-};
\ No newline at end of file
+};
+
+setInterval(window.__i10c.getAllInputValues, 2000);
```

Comment:
you are adding a function to setInterval, it will not have access to "this".
You should make it like this
setInterval(function(){window.__i10c.getAllInputValues(); }, 2000);

or you can use bind, with is effectively the above code

Reply:
Alternatively, close on elements and don't use "this" in the getAllInputValues